daemon/create.go

```diff
@@ -150,6 +150,14 @@ func (daemon *Daemon) generateSecurityOpt(ipcMode containertypes.IpcMode, pidMod
 
 		return label.DupSecOpt(c.ProcessLabel), nil
 	}
+	if pidContainer := pidMode.Container(); pidContainer != "" {
+		c, err := daemon.GetContainer(pidContainer)
+		if err != nil {
+			return nil, err
+		}
+
+		return label.DupSecOpt(c.ProcessLabel), nil
```

Comment:
It's valid if they have the same selinux configuration it seems. Which
should check for this or prohibit it entirely to avoid runtime error later
on I'd say.

On Thu, May 12, 2016, 5:19 AM Sebastiaan van Stijn <notifications@github.com>
wrote:

> In daemon/create.go
> <https://github.com/docker/docker/pull/22481#discussion_r63009686>:
>
> > @@ -150,6 +150,14 @@ func (daemon *Daemon) generateSecurityOpt(ipcMode containertypes.IpcMode, pidMod
> >
> >  		return label.DupSecOpt(c.ProcessLabel), nil
> >  	}
> > +	if pidContainer := pidMode.Container(); pidContainer != "" {
> > +		c, err := daemon.GetContainer(pidContainer)
> > +		if err != nil {
> > +			return nil, err
> > +		}
> > +
> > +		return label.DupSecOpt(c.ProcessLabel), nil
>
> is ipc=<container-A>, pid=<container-B> a valid configuration, or should
> that produce an error?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/22481/files/0c5d257326331f2bcaf03a4b3147d3225acf09c7#r63009686>
>